packages/create-sitecore-jss/src/templates/nextjs-xmcloud/src/lib/next-config/plugins/feaas.js

```diff
@@ -3,6 +3,19 @@
  */
 const feaasPlugin = (nextConfig = {}) => {
   return Object.assign({}, nextConfig, {
+    // Enable FEAAS Nextjs Image integration
+    images: {
+      remotePatterns: [
+        {
+          protocol: 'https',
+          hostname: '**',
+        },
+        {
+          protocol: 'http',
+          hostname: '**',
```

Comment:
@Inviz You are talking about feaas + NextImage integration? It's a different item that we discussed, right?

Reply:
This config should only be concerned with external image sources that we (Sitecore) own/control, right? If a customer is using other external sources then they would add their own remotePatterns.

Reply:
Yes, but what's important is that the next.js's behavior for when hostname is not in the whitelist is simply to not show any image. We can not know upfront on our end if it'll work or not. We can not afford breaking images in user components. They may want to optimize images linked from other sites, or coming from CMSes and such. But evne if we use strict whitelist, we should ensure we are not breaking the user image.  The only way to do it without whitelisting everything is:
1) Either find a way that Next.js would throw error when NextImage recieves unwhitelisted src
2) Pass `unoptimized` flag to NextImage to be set to true, when src is detected not to be whitelisted. This would have to be done in the JSSImage by inspecting the SRC and running it against patterns manually.

Reply:
INVALID_IMAGE_OPTIMIZE_REQUEST is thrown when SRC is not whitelisted.

Reply:
You are right
I guess we need to consider this problem as a part of different work item, since NextImage is not yet used with FEaaS
We are discussing this problem. I close this comment

Reply:
@Inviz It's a problem not only related to FEAAS but even our JSSImage, after we introduced nextjs image optimization